Wrapping/Generators/Python/itk/__init__.py

```diff
@@ -18,9 +18,13 @@
 
 """itk: Top-level container package for ITK wrappers."""
 
-from .conf import itkConfig as _itkConfig
+from .conf.itkConfig import ITK_GLOBAL_VERSION_STRING as __version__
 
-__version__ = _itkConfig.ITK_GLOBAL_VERSION_STRING
+from .conf.itkTemplate import image
+from .conf.itkTemplate import output
+from .conf.itkExtras import *
+from .conf.itkInitHelpers import *
+from itkTypes import *
```

Comment:
NOTE:  Much of the work up until now was a result of changes made in search of the mechanisms needed to provide the above "standard" symbol lifting behavior that mypy and pytype can interpret.  

This is a key patch-set change on the way to automating the introspection capabilities that are desired.